src/ansys/aedt/core/modeler/circuits/primitives_maxwell_circuit.py

```diff
@@ -284,3 +284,87 @@ def create_winding(self, name=None, location=None, angle=0, use_instance_id_netl
         )
         id.set_property("Name", name)
         return id
+
+    @pyaedt_function_handler(compname="name")
+    def create_isin(self, name=None, value=1, location=None, angle=0, use_instance_id_netlist=False):
```

Comment:
@DaveTwyman I'd change name: create_i_sin and create_v_sin. This is more pythonic.

Reply:
@gmalinve , Yes ok I think your suggestion makes it easier to read than 'create_isin' and 'create_vsin'. I'll change this

Reply:
I've changed this so marked as resolved